.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,30 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/docker-in-docker
+{
+	"name": "CPA",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"build": {		
+        "context": "..",
+		"dockerfile": "Dockerfile"
+	},	
+
+	"features": {
+		"ghcr.io/devcontainers/features/docker-in-docker:2": {
+			"version": "latest",
+			"enableNonRootDocker": "true",
+			"moby": "true"
+		}
+	},
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"runArgs": ["--env-file",".devcontainer/.env"],
+
+	// Configure tool-specific properties.
+	// "customizations": {},
```

Comment:
Can we add installation of `golang.go` extension?

Reply:
Done

Reply:
Added